examples/baselines/2dhistos/README.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
If you are gonna have a readme, put something in it at least. It is fine if you don't finish but put something.

Reply:
Fixed